docs/configuration/http_conn_man/route_config/traffic_shifting.rst

```diff
@@ -0,0 +1,57 @@
+.. _config_http_conn_man_route_table_traffic_shifting:
+
+Traffic Shifting
+================
+
+Envoy's router can accomplish traffic shifting between routes of a virtual host
+based on the :ref:`runtime <config_http_conn_man_route_table_route_runtime>` object in the route configuration.
+A common use case is shifting traffic between clusters with different versions of a service deployed on them.
+
+.. code-block:: json
+
+    {
+      "route_config": {
+        "virtual_hosts": [
+          {
+            "name": "service",
+            "domains": ["*"],
+            "routes": [
+              {
+                "prefix": "/",
+                "cluster": "service_v1",
+                "retry_policy": {
```

Comment:
let's simplify the example and remove all of the extraneous options like retry_policy and host_rewrite.